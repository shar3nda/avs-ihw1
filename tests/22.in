6 10 6 -5 7 -3 6 0 7 -3