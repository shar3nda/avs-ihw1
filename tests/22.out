-5 7 -3 6 0 7 -3